src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -110,16 +110,73 @@ function dispatchLayersFromUrlIntoStore(store, urlParamValue) {
                 )
             }
             log.debug(`  Add layer ${parsedLayer.id} to active layers`, layerObject)
+            if (layerObject.customAttributes?.features) {
+                layerObject.customAttributes.features
+                    .toString()
+                    .split(':')
+                    .forEach((featureId) => {
+                        featuresRequests.push(
+                            getFeature(
+                                store.getters.getLayerConfigById(parsedLayer.id),
+                                featureId,
+                                store.state.position.projection,
+                                store.state.i18n.lang
+                            )
+                        )
+                    })
+            }
         }
         return layerObject
     })
+
     promisesForAllDispatch.push(
         store.dispatch('setLayers', { layers: layers, dispatcher: STORE_DISPATCHER_ROUTER_PLUGIN })
     )
+    if (featuresRequests.length > 0) {
+        promisesForAllDispatch.push(getAndDispatchFeatures(featuresRequests, store)) // all features after promise
```

Comment:
the comment on this line doesn't make much sense to me.
could you rephrase it, or remove it?